### DISCUSSION TITLE
how to disable the feature of  parameterized endpoint grouping

### DISCUSSION THREAD

**Discussion by szhengli** [apache/skywalking#9692] [Q&A]
I need to disable   parameterized endpoint grouping. anyone knows ? thanks

**Comment by wu-sheng** [comment]
It is never activated by default.

**Reply by szhengli** [reply]
as below shows, all of these are auto grouped. any way to disable the auto grouping, instead show the full api url ? 

![image](https://user-images.githubusercontent.com/16127498/192541521-b4b55e35-dcda-4834-8bbe-73d41811b6ec.png)

**Reply by szhengli** [reply]
just add another shot;  
![image](https://user-images.githubusercontent.com/16127498/192543025-0d5cff90-cf98-45ae-8e5b-de8ec23cb6c5.png)

**Reply by wu-sheng** [reply]
That is not grouping. 
That is SpringMVC plugin's feature. That name is reported by the plugin directly. OAP do nothing.

**Reply by wu-sheng** [reply]
This is how your codes write(pattern match in Spring Controller), and IP:port is never recommended in the endpoint.

**Reply by szhengli** [reply]
I'm ops, not much in java coding. I will consult with dev  tomorrow.   Your repsonse is very informative, much appreciated. cheers !
